etl/steps/data/garden/oecd/2023-05-19/co2_air_transport.py

```diff
@@ -0,0 +1,307 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import numpy as np
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("co2_air_transport.start")
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow: Dataset = paths.load_dependency("co2_air_transport")
+    ds_tour: Dataset = paths.load_dependency("unwto")
+
+    # Read table from meadow dataset.
+    tb_meadow = ds_meadow["co2_air_transport"]
+    tb_tour = ds_tour["unwto"]
+    df = pd.DataFrame(tb_meadow)  # Create a dataframe with data from the co2 transport table.
+    df_tr = pd.DataFrame(tb_tour)  # Create a dataframe with data from the tourism table.
+
+    # list of months
+    month_names = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December",
+    ]
+
+    #
+    # Process data.
+    #
+    log.info("co2_air_transport.harmonize_countries")
+
+    df = geo.harmonize_countries(df=df, countries_file=paths.country_mapping_path)
+    # Only use passenger flights and total international/domestic emissions
+    df = df[df["flight_type"] == "P"]
+    df.drop("flight_type", axis=1, inplace=True)
+    df = df[df["emission_source"].isin(["TER_DOM", "TER_INT"])]
+    df = df.reset_index(drop=True)
+
+    # Process annual data
+    pivot_table_ye = process_annual_data(df)
+
+    # Add population data to the DataFrame
+    pivot_table_ye = geo.add_population_to_dataframe(
+        pivot_table_ye, country_col="country", year_col="year", population_col="population"
+    )
+
+    emissions_columns = pivot_table_ye.columns[2:-1]
```

Comment:
This is a bit cryptic - might be better to explicitly do `emissions_columns = ['TER_DOM_a', 'TER_INT_a']` or this `emissions_columns = [col for col in emissions_columns if col not in ("country", "year", "population")]`